pom.xml

```
@@ -358,14 +428,13 @@
         <groupId>org.codehaus.mojo</groupId>
         <artifactId>buildnumber-maven-plugin</artifactId>
       </plugin>
+
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
-        <version>3.0.0-M5</version>
-        <configuration>
-          <argLine>${surefireJvmArgs}</argLine>
-        </configuration>
+        <version>2.22.2</version>
```

Why use such an old version of the plugin???
```suggestion
        <version>${surefire-version}</version>
```
and set property `surefire-version = 3.0.0-M7`